src/estimation/src/GravityCompensationHelpers.cpp

```diff
@@ -0,0 +1,186 @@
+#include "iDynTree/Estimation/GravityCompensationHelpers.h"
```

Comment:
The copyright header is necessary.

Reply:
Addded copyright header and newline in this commit: https://github.com/robotology/idyntree/pull/438/commits/fd9efa9536bf836259f09b65ae8693a43dddd1f7